_posts/2020-09-21-permission-editor.md

```diff
@@ -0,0 +1,111 @@
+
+The default settings are the following:
+
+```{
+   "groups":[
+      {
+         "Read":["See", "Preview", "PreviewWithoutWatermark", "PreviewWithoutRedaction", "Open"]
+      },
+      {
+         "Write":[ "Save", "AddNew", "Delete"]
+      },
+      {
+         "Versioning":["OpenMinor", "Publish", "ForceCheckin", "Approve", "RecallOldVersion"]
+      },
+      {
+         "Advanced":["SeePermissions", "SetPermissions", "RunApplication"]
+      }
+   ]
+}
+```
+
+On the permission editor dialog you can see the groups and permissions based on these settings. 
+
+<p align="center">
+<img src="/img/posts/permission_groups.png">
+</p>
+
+## How to use it? 🎮
+
+Select a content from your repository, right-click on it and select 'Set permissions' from the dropdown. On the permission view you can check the permission entries (users or groups) which are inherited or set directly on the current content. Selecting an entry makes the permission editor dialog open.
+
+On the dialog use the switchers to enable/disable a permission. There are also group level switchers, so you can easily enable/disable all the permissions in the selected group...and there is the king of kings: Full access 👑 with which you can turn all permissions on/off.
+
+Some permissions are disabled because they are inherited and you cannot override them. The changes only apply on clicking Submit button so for your comfort there is also a reset button to restore the unsaved changes.
+
+<p align="center">
+<img src="/img/posts/permission_how_to_use.gif">
+</p>
+
+## Additional features 💄👓👜
+
+**Make content public/private**
+
+If you would like to share your content (content can be almost anything, cause in sensenet everything is a content) with somebody e.g. from another company who has no authority to your repository you can do it easily by clicking on "Make content public" button. If you change your mind reverting this is easy as it was to set.
+
+<p align="center">
+<img src="/img/posts/permission_make_public.gif">
+</p>
+
+**Assign new permission**
+
+If you would like to grant permission for someone on a content who has no permission setting at all, you can do it using the _Assign new permission_ button. Search for and select the user or group to whom you want to set permissions and turn on/off the switchers. 😉
+
+<p align="center">
+<img src="/img/posts/permission_assign_new.gif">
+</p>
+
+**Local only**
+
+One of the big advantages of sensenet content repository tree is inheritance that is applied on permissions as well. However, there are cases when you do not want child contents to inherit permissions. For example you want to allow certain users to see a content (e.g. a Content List) but you do not want them to be able to see contents that are inside this container. To avoid permission inheritance, mark permission entries as 'local-only'.
```

Comment:
```suggestion
One of the big advantages of sensenet content repository tree is inheritance which is applied on permissions as well. However, there are cases when you do not want child contents to inherit permissions. For example you want to allow certain users to see a content (e.g. a Content List) but you do not want them to be able to see contents that are inside this container. To avoid permission inheritance, mark permission entries as 'local-only'.
```